examples/medplum-demo-bots/src/deduplication/README.md

```diff
@@ -0,0 +1,56 @@
+# Deduplication Demo
```

Comment:
Make it clear that we're focusing on _patient_ deduplication. 

Good link below to the architecture doc.